Update zha.markdown with info on visualization

Hedda opened the pull request:
## Proposed change

Update ZHA integration documentation with basic info on the visualization window/tab as a seed for what is requested in https://github.com/home-assistant/home-assistant.io/issues/26300

Please note that this only provides some initial basic information that probably needs to be fleshed out further by ZHA developers.

That is, I do not think that the initial commit in this PR alone currently provides all the information that is asked for in https://github.com/home-assistant/home-assistant.io/issues/26300

Still, hope that this is good enough for now so that can later be built out by others,

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Hedda committed:
Update zha.markdown with info on visualization

Update ZHA integration documentation with seed info on the visualization window/tab as requested in https://github.com/home-assistant/home-assistant.io/issues/26300

Please note that this only provides some initial basic information that may need to be fleshed out further by ZHA developers.

dannytsang commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+The visualization tried to show multi-hop connections between your paired devices and their reported cumulative values of RSSI (“Received Signal Strength Indicator") and LQI ("Link Quality Indication").
+
+Note that the exact method in which these values are reported depends on the Zigbee network stack being used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```
Replace this with the <div class='note'></div> standard note section.

Hedda replied:
@dannytsang  standard note section? No other notes in ZHA integration documentation use such, can be seperate PR?

dannytsang replied:
Yeah... fair enough. Don't forget to mark it ready for review when you're ready.

Hedda replied:
OK thank you for the review. I will mark this conversation as resolved and then mark this PR ready for review again.

dannytsang commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+This network visualization can sometimes be of help to get a quick overview in identifying devices that are reporting links with poor connection (e.i. low values), however, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
+
+The visualization tried to show multi-hop connections between your paired devices and their reported cumulative values of RSSI (“Received Signal Strength Indicator") and LQI ("Link Quality Indication").
```
```suggestion
The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
```

dannytsang commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+ZHA configuration UI has a window/tab for "Visualization" that provides a visual representation of device links in your Zigbee network topology.
+
+This network visualization can sometimes be of help to get a quick overview in identifying devices that are reporting links with poor connection (e.i. low values), however, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
```
```suggestion
The network visualization can help to get a quick overview in identifying devices that are reporting links with poor connection (i.e. low values). However, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
```

dannytsang commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
 
+### Zigbee network visualization in ZHA UI
+
+ZHA configuration UI has a window/tab for "Visualization" that provides a visual representation of device links in your Zigbee network topology.
```
```suggestion
ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.
```
Use Markdown syntax for highlighting words.

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Danny Tsang <567982+dannytsang@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Danny Tsang <567982+dannytsang@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Danny Tsang <567982+dannytsang@users.noreply.github.com>

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
 
+### Zigbee network visualization in ZHA UI
+
+ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.
```
```suggestion
ZHA configuration UI has a window/tab for *Visualization* that visually represents device links in your Zigbee network topology.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.
+
+The network visualization can help to get a quick overview in identifying devices that are reporting links with poor connection (i.e. low values). However, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
```
```suggestion
The network visualization can help get a quick overview in identifying devices reporting links with poor connection (that is, low values). However, be aware that you will need to look at the ZHA logs instead for deeper troubleshooting.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
   - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
   - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.
 
+### Zigbee network visualization in ZHA UI
+
+ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.
+
+The network visualization can help to get a quick overview in identifying devices that are reporting links with poor connection (i.e. low values). However, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
+
+The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
+
+Note that the exact method in which these values are reported depends on the Zigbee network stack being used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```
you can drop `being` from `network stack being used`

Hedda replied:
OK, done.

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Hedda committed:
Update zha.markdown

Hedda committed:
Merge branch 'current' into patch-85

frenck reviewed (commented):
Reading this part, I think the biggest question on this isn't answered. What is a high/log/normal RSSI or LQI value?

Hedda commented:
> Reading this part, I think the biggest question on this isn't answered. What is a high/log/normal RSSI or LQI value?

@puddly Do you have any suggestions here? I guess that explaining RSSI can be done relatively straightforward(?) but explaining LQI is much more complicated since it will depend on Zigbee stack devices are using? As discussed here -> https://github.com/zigpy/zigpy/issues/886

RSSI (Received Signal Strength Indicator) are shown as negative numbers represented in -dBm format (0 to -100) and is used as an indicator of the raw signal strength between two devices. 
* Generally, anything -60 and above (meaning -50, -40, etc.) in RSSI should be considered a strong signal (not losing messages). 
* Anything at -80 and below ( meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.

It is also not recommended to judge RSSI and LQI on their own separately as it is really the combination of those two that you need to interpret when troubleshooting if Zigbee messages are being dropped.

As an example, Hubitat developers recommend that their Zigbee end-users look at "lastHopLqi" (Last hop LQI) in their Zigbee logs when troubleshooting unstable connections in a Zigbee network mesh:

https://community.hubitat.com/t/zigbee-signal-strength/5012

https://docs2.hubitat.com/en/user-interface/settings/zigbee-logs

* _**lastHopLqi** - is an unsigned 8-bit integer ranging from 0 to 255, with the maximum value representing the best possible link quality index (LQI)( for delivery of the current packet across the last hop of the incoming route. LQI data is also used by the Zigbee stack to determine connectivity between neighboring nodes, and to select parent devices when joining the network as a ZigBee end device. An LQI below 200 represent links with high error rates, while and LQI of 200 represents approximately an 80% reliability of receiving the data packet intact._

* _**RSSI** or Receiver Signal Strength (**B**) - is a signed 8-bit integer ranging from approximately -100 to 127, with each value representing the energy level in dBm or decibel-milliwatts at the radio receiver. Indication is the signal strength. Closer to 0 the better as that means less power was used to transmit and is an indication of stronger signal strength. This may be helpful to locate a Zigbee Endpoint device that is dropping off due to interference or weak signal. Since RSSI information does not pertain to any specific link with another device, the measured signal energy at a given frequency can come from any transmitter or device causing interference on that frequency, whether it is another nearby node on the same network, a device from a different PAN (Personal Area Network) on the same channel, or a non-ZigBee, non-802.15.4 source of interference such as a WiFi transmitter or microwave. However, it is important to remember that RSSI information is only between the Endpoint device and the first hop from that device (nearest router / repeater), so this may not necessarily be the hub._

![image](https://user-images.githubusercontent.com/6320001/229098596-09cb6604-9c55-4e39-95d7-3d91557dad76.png)

frenck commented:
Not sure why this was marked as ready for review.

Hedda commented:
> Not sure why this was marked as ready for review.

I think it ready as-is. I do not believe that the question "**_What is a high/low/normal RSSI or LQI value?_**" can easily be answered.

Unless ZHA/zigpy developer(s) can help answer that question I do not know how to summarise a short and concise answer.

I could simply add an explanation of LQI is, but do not think I can give a general statement of what high/low/normal value is.

As I wrote in my original post, I know that this does not cover all aspects, but I think it is better than nothing like is there now.

Hedda committed:
ZHA interpret RSSI and LQI

How to interpret RSSI and LQI values

Hedda commented:
> Reading this part, I think the biggest question on this isn't answered. What is a high/log/normal RSSI or LQI value?

OK I have now updated this PR with a subsection on "How to interpret RSSI and LQI values" which I now hope is better than none.

Hedda committed:
Update zha.markdown lint

  600:1-600:129  warning  Marker style should be `-`  unordered-list-marker-style  remark-lint
  601:1-601:125  warning  Marker style should be `-`  unordered-list-marker-style  remark-lint
  605:1-605:185  warning  Marker style should be `-`  unordered-list-marker-style  remark-lint

Hedda committed:
Merge branch 'current' into patch-85

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
 
+### Zigbee network visualization in ZHA UI
+
+ZHA configuration UI has a window/tab for *Visualization* that visually represents device links in your Zigbee network topology.
```
```suggestion
The ZHA configuration UI has a tab to visualize device links in your Zigbee network topology.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+ZHA configuration UI has a window/tab for *Visualization* that visually represents device links in your Zigbee network topology.
+
+The network visualization can help get a quick overview in identifying devices reporting links with poor connection (that is, low values). However, be aware that you will need to look at the ZHA logs instead for deeper troubleshooting.
```
```suggestion
The network visualization can help to identify devices with poor connection (that is, low values on the link). To find more detailed information required for troubleshooting, you will need to look at the ZHA logs.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
+
+Note that the exact method in which these values are reported depends on the Zigbee network stack used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```
``` suggestion
The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+#### How to interpret RSSI and LQI values
+
+Interpreting RSSI and LQI values is complicated and the recommendation is that normal users should ignore them unless guided by Zigbee troubleshooting specialists. If you do delve into this then it is important to understand not to judge only RSSI or LQI values on their own separately as it is really the combination of these two that needs to interpret be when troubleshooting if Zigbee messages a ...
```
```suggestion
Interpreting RSSI and LQI values is complex. Unless you are Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading.  If you do delve into this, then it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+Interpreting RSSI and LQI values is complicated and the recommendation is that normal users should ignore them unless guided by Zigbee troubleshooting specialists. If you do delve into this then it is important to understand not to judge only RSSI or LQI values on their own separately as it is really the combination of these two that needs to interpret be when troubleshooting if Zigbee messages a ...
+
+RSSI (Received Signal Strength Indicator) values are simple to interpret as they are just an indicator value of the raw signal strength between two devices. RSSI values are shown as negative numbers represented in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values are better as they indicate less interference and a good signal. ...
```
```suggestion
RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+RSSI (Received Signal Strength Indicator) values are simple to interpret as they are just an indicator value of the raw signal strength between two devices. RSSI values are shown as negative numbers represented in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values are better as they indicate less interference and a good signal. ...
+
+- Generally, anything -60 and above (meaning -50, -40, etc.) in RSSI should be considered a strong signal (not losing messages).
+- Anything at -80 and below ( meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.
```
```suggestion
- Anything at -80 and below (meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+- Generally, anything -60 and above (meaning -50, -40, etc.) in RSSI should be considered a strong signal (not losing messages).
+- Anything at -80 and below ( meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.
+
+LQI (Link Quality Index) values can be hard to interpret for Zigbee. The reason for this is that how to perform LQI measurements are not standardized as part of the Zigbee specifications (and the IEEE 802.15.4 specification). LQI values are shown as positive numbers on a scale, but because the information value provided by the Zigbee devices is not measured using unified standards from all device ...
```
```suggestion
LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks the values can not always be trusted. For example, Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse. The Texas Instruments Z-Stack computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio, that more or less just provide an LQI value that based on the strength of the received signal.  This can be misleading in  case the user have a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). And other manufacturers and Zigbee stacks measure and calculate LQI values in another way.
```

c0ffeeca7 commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+LQI (Link Quality Index) values can be hard to interpret for Zigbee. The reason for this is that how to perform LQI measurements are not standardized as part of the Zigbee specifications (and the IEEE 802.15.4 specification). LQI values are shown as positive numbers on a scale, but because the information value provided by the Zigbee devices is not measured using unified standards from all device ...
+
+- LQI value 255 means a zero error rate in theory, a positive high LQI value is better and a lower LQI value is worse, however, depending on your devices that might not be the reality.
```
```suggestion
- In theory, an LQI value 255 means a zero error rate in theory. And in general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices that might not be the reality.
```

c0ffeeca7 reviewed (commented):
Some style-related suggestions.

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Hedda commented:
> Some style-related suggestions.

@c0ffeeca7 Thanks, all suggested changes are applied now.

c0ffeeca7 reviewed (dismissed):
Thank you, @Hedda :+1: LGTM.

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+#### How to interpret RSSI and LQI values
+
+Interpreting RSSI and LQI values is complex. Unless you are Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading.  If you do delve into this, then it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.
```
```suggestion
Interpreting RSSI and LQI values is complex. Unless you are a Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading. If you delve into this, it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+The ZHA configuration UI has a tab to visualize device links in your Zigbee network topology.
+
+The network visualization can help to identify devices with poor connection (that is, low values on the link). To find more detailed information required for troubleshooting, you will need to look at the ZHA logs.
```
```suggestion
The network visualization can help to identify devices with poor connection (that is, low values on the link). You will need to look at the ZHA logs to find more detailed information required for troubleshooting.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
+
+The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```
```suggestion
The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the message propagates through the mesh networking matrix.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+Interpreting RSSI and LQI values is complex. Unless you are Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading.  If you do delve into this, then it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.
+
+RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from  ...
```
```suggestion
RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such, it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+- Generally, anything -60 and above (meaning -50, -40, etc.) in RSSI should be considered a strong signal (not losing messages).
+- Anything at -80 and below (meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.
+
+LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks the  ...
```
```suggestion
LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks, the values can not always be trusted. For example, Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse. The Texas Instruments Z-Stack computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio that more or less just provides an LQI value that, based on the strength of the received signal.  This can be misleading in case the user has a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). Other manufacturers and Zigbee stacks measure and calculate LQI values in another way.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
+
+LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks the  ...
+
+- In theory, an LQI value 255 means a zero error rate in theory. And in general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices that might not be the reality.
```
```suggestion
- In theory, an LQI value of 255 means a zero error rate in theory. In general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices, that might not be the reality.
```

frenck committed:
Tiny language tweaks

frenck reviewed (approved):
Thanks all! 👍 

../Frenck

c0ffeeca7 reviewed (approved):
LGTM :+1:
